frontend/package.json

```diff
@@ -9,12 +9,16 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "next": "15.0.2",
+    "next-auth": "^5.0.0-beta.25",
     "react": "19.0.0-rc-02c0e824-20241028",
-    "react-dom": "19.0.0-rc-02c0e824-20241028",
-    "next": "15.0.2"
+    "react-dom": "19.0.0-rc-02c0e824-20241028"
   },
   "devDependencies": {
+    "@types/node": "22.9.0",
+    "@types/react": "18.3.12",
     "postcss": "^8",
-    "tailwindcss": "^3.4.1"
+    "tailwindcss": "^3.4.1",
+    "typescript": "5.6.3"
```

Comment:
Yes, I used to use ts only for the [...nextauth]/routes.ts file, which comes from the NextAuth documentation. I kept it in ts to ensure the NextAuth functions work correctly. But since I'm not using NextAuth and any ts in the latest PR, I should remove the ts packages of course.